### PR TITLE
Update install

### PR DESCRIPTION
**Pull request by gillian-earthscope**
* fixes compile issue with XCode 16
 * updates installation to R=4.2 and other package version updates
 * transitions service.iris.edu to service.earthscope.org
 * changes http to https

### CODE REVIEW

File: ispaq/PSD_metrics.py [hunk 1]
```diff
@@ -291,7 +291,7 @@ def do_pdf(concierge, starttime, endtime):
         try:
             resp_url = Client("IRIS")
         except Exception as e:
-            logger.error("Could not connect to 'http:/service.iris.edu/irisws/evalresp/1'")
+            logger.error("Could not connect to 'https:/service.earthscope.org/irisws/evalresp/1'")
```
Comment (laura-iris) [inline ispaq/PSD_metrics.py:294]: the browser will redirect, but might want to change to https://service.earthscope.org/irisws/evalresp/1 anyway

File: ispaq/transferFunction_metrics.py [hunk 2]
```diff
@@ -62,7 +62,7 @@ def transferFunction_metrics(concierge):
         try:
             resp_url = Client("IRIS")
         except Exception as e:
-            logger.error("Could not connect to 'http:/service.iris.edu/irisws/evalresp/1'") 
+            logger.error("Could not connect to 'https:/service.earthscope.org/irisws/evalresp/1'")
```
Comment (laura-iris) [inline ispaq/transferFunction_metrics.py:65]: same here about the url ://

**Review by laura-iris — COMMENTED** [review]
Looks good - just a couple very simple comments. 

Also, do we want to add EarthScope as an alias that can be used in the preference file. We would have to convert it to IRIS before we pass it along to the obspy URL_MAPPINGS, since that doesn't include earthscope references yet. But that way users would be able to start using EarthScope in the preference file, if they are so inclined.

**Review by laura-iris — COMMENTED** [review]
Looks good!

**Comment by gillian-earthscope** [comment]
Added EARTHSCOPE alias, updated documentation, and made explicit that events go to the USGS event service when using EarthScope data

**Review by laura-iris — APPROVED** [review]

**Review by laura-iris — APPROVED** [review]

**Pull request merged**